### PR TITLE
Created header conmponent

### PR DESCRIPTION
**Pull request by mwangimuringi**
A navbar header which has logo, search bar and header links.
 its responsive.

### CODE REVIEW

File: client/package.json
Comment (muringitech) [inline client/package.json:1]: React icons are imported well

File: client/src/App.jsx
Comment (muringitech) [inline client/src/App.jsx:1]: Brilliant

File: client/src/components/Header.jsx
Comment (muringitech) [inline client/src/components/Header.jsx:1]: Job well done

File: client/src/pages/About.jsx
Comment (muringitech) [inline client/src/pages/About.jsx:1]: Ok

**Review by muringitech — COMMENTED** [review]
Its good

File: client/package.json
Comment (mwangimuringi) [reply client/package.json:1]: ill merge the branch

File: client/src/App.jsx
Comment (mwangimuringi) [reply client/src/App.jsx:1]: good

File: client/src/components/Header.jsx
Comment (mwangimuringi) [reply client/src/components/Header.jsx:1]: nice

**Pull request merged**